### ISSUE TITLE
Import original csv sorting without sorting field

### ISSUE THREAD

**Issue by glucka** [cobwebch/external_import#111]
It is possible import the csv sorting without sorting field in csv? 
With import i mean to fill a DB sorting field with the original csv row indexes.
Or exists a simple way to empty the DB befor every import?

**Comment by fsuter** [comment]
No, it is currently not possible to define the sorting field and there's also no simple way to empty th database. Both can be achieved by programming custom steps, but that does not qualify as "simple" ;-)

The point about sorting is very interesting though and it could be a feature to add in the future.

**Comment by glucka** [comment]
Thanks for the answer. I'm trying with custom steps!

**Comment by fsuter** [comment]
Thanks for cleaning up ;-) I'll leave this issue open, however, as a reminder that a feature for sorting could be useful.
